tests/experimental/nn/test_sync_batchnorm.py

```diff
@@ -36,40 +36,48 @@ def wrapper(*args, **kwargs):
 
 
 def check_parity(torch_bn, fs_bn, x):
-    yh = torch.ones_like(x)
-    torch_y = torch_bn(x)
-    fs_y = fs_bn(x)
+    yh = torch.randn_like(x)
+    torch_x = x.detach()
```

Comment:
why do you need to detach input here?

Reply:
Because I want `torch_x` and `fs_x` to share the same data but have different gradients. I guess I could have used `clone()` instead and just created multiple copies of the `x` tensor.

Reply:
I thought detach removes the variable from the graph but I guess you then specify requires_grad which makes it a part of the graph again. Either way is fine with me. I guess I was not familiar with the usage.